air/src/proof/ood_frame.rs

```diff
@@ -149,13 +194,68 @@ impl Deserializable for OodFrame {
         let num_trace_state_bytes = source.read_u16()? as usize;
         let trace_states = source.read_vec(num_trace_state_bytes)?;
 
+        // read lagrange kernel column trace rows
+        let num_lagrange_state_bytes = source.read_u16()? as usize;
+        let lagrange_kernel_trace_states = source.read_vec(num_lagrange_state_bytes)?;
+
         // read constraint evaluations row
         let num_constraint_evaluation_bytes = source.read_u16()? as usize;
         let evaluations = source.read_vec(num_constraint_evaluation_bytes)?;
 
         Ok(OodFrame {
             trace_states,
+            lagrange_kernel_trace_states,
             evaluations,
         })
     }
 }
+
+// OOD FRAME TRACE STATES
+// ================================================================================================
+
+/// Stores the trace evaluations evaluated at `z` and `gz`, where `z` is a random Field element. If
```

Comment:
nit: we should probably say: "... trace evaluations at ..."

Reply:
Agreed, fixed